teams/project-unsafe-thir-refactor.toml

```diff
@@ -0,0 +1,14 @@
+name = "project-unsafe-thir-refactor"
```

Comment:
Maybe this could be renamed to `project-thir-unsafeck` or something? The current name makes it sound like it's refactoring the THIR in a dangerous way.

Reply:
heh, true :) I'm up for whatever name.

Reply:
```suggestion
name = "project-thir-unsafeck"
```

Reply:
(Probably want to rename the repo as well.)